gitlab-status/handler.go

```diff
@@ -112,3 +145,55 @@ func validateRequest(req []byte) (err error) {
 
 	return nil
 }
+
+func FormatLogsURL(gatewayURL string, event *sdk.Event) (string, error) {
```

Comment:
Curious why these methods are exported (starting with capital letters)?

Reply:
Though they would be needed in the future to be added to SDK but I will probably revert that

Reply:
now the methods are not exported